Examples/zib-Pregnancy-Duration - example.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/observation.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<Observation xmlns="http://hl7.org/fhir" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/STU3/observation.xsd">
+    <id value="zib-pregnancy-pregnancyduration-example"/>
+    <meta>
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-Pregnancy-PregnancyDuration"/>
+    </meta>
+    <extension url="http://hl7.org/fhir/StructureDefinition/event-partOf">
+        <valueReference>
+            <reference value="Condition/zib-pregnancy-example" />
+            <display value="Example pregnancy" />
+        </valueReference>
+    </extension>
+    <status value="final"/>
+    <code>
+        <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="57036006"/>
+            <display value="Zwangerschapsduur"/>
+        </coding>
+    </code>
+    <subject>
+        <reference value="Patient/example"/>
+        <display value="Example Patient"/>
+    </subject>
+    <effectiveDateTime value="2014-09-18" />
+    <valueQuantity>
+        <value value="24" />
+        <system value="http://unitsofmeasure.org" />
+        <code value="wk" />
```

Comment:
Het lijkt me eigenlijk verstandiger alleen met dagen  te werken. De zib zegt inderdaad 'De duur kan in dagen (d) of weken (wk) opgegeven worden.' maar dit vereist van alle leveranciers wel dat ze dit allebei omrekenen. Wat de zorgverlener wil zien is in verloskunde altijd w/d, dus 14/4 is '14 weken en 4 dagen'. Opgeven in FHIR in 'w' werkt dan alleen bij gehele weken, en specifiek voor deze casus zou dan '10 w' ingevoerd worden i.p.v. het even geldige '70 d'.
Heb ook WV in BITS gezet voor de ZIB: https://bits.nictiz.nl/browse/ZIB-986

Reply:
@ArdonToonstra hoe denk jij hierover? Afwijken van de zib in dit geval?